dali/pipeline/pipeline.h

```diff
@@ -89,13 +89,23 @@ class DLL_PUBLIC Pipeline {
                       bool async_execution = true, bool dynamic_execution = false,
                       size_t bytes_per_sample_hint = 0, bool set_affinity = false);
 
+  /**
+   * @warning This constructor is deprecated. Use
+   *          Pipeline(const string &serialized_pipe, const PipelineParams &param_override) instead.
+   */
```

Comment:
Would it make sense to remove this ctor instead of marking it as deprecated?

Reply:
It would, but it would generate a lot of noise in the tests, so I keep this change for a separate future PR.